src/app/payment/paymentMethod/MolliePaymentMethod.tsx

```diff
@@ -68,17 +69,22 @@ const MolliePaymentMethod: FunctionComponent<MolliePaymentMethodsProps> = ({ ini
         return method.method === MolliePaymentMethodType.creditcard;
     }
 
-    return <>
+    function validateInstrument() {
+        return <></>;
```

Comment:
what is the purpose of this empty function?

Reply:
To not show the default component "CVC" that validates the vaulted credit card

Reply:
This looks like a code smell to me. Passing a function that renders an empty element doesn't seem very elegant. Looking at the HostedWidgetPaymentMethod props there seems to be a specific property for this? `hideVerificationFields={ true }` rather than passing an empty validateInsutrment function ?

Reply:
yes, you are right!

by the way now i remember why i added a ternary on the MollieCustomCardForm

![Screen Shot 2021-04-08 at 10 29 28](https://user-images.githubusercontent.com/69221626/114055160-1d3bcd80-9856-11eb-9465-bcfc66d4827b.png)